### PR TITLE
Bare metal kb update

### PR DESCRIPTION
**Pull request by chrislittle**

### CODE REVIEW

**Comment by chrislittle** [comment]
unsure why this is also failing with just 1 sentence changed.  logs don't really show a clear reason to me.

**Comment by bryanfriedman** [comment]
Thanks @chrislittle...looks good.

File: Servers/creating-a-new-bare-metal-server.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {{{
   "title": "Creating a New Bare Metal Server",
-  "date": "7-5-2015",
+  "date": "3-14-2016",
   "author": "Bryan Friedman",
   "attachments": []
```
Comment (jim-phillips) [inline Servers/creating-a-new-bare-metal-server.md]: @chrislittle You are missing a comma at the end of the attachments line.  `[],`

**Comment by chrislittle** [comment]
all set.  I didn't check the previous code

**Pull request merged**